packages/ci/test/commands/ci/index.test.ts

```diff
@@ -35,31 +35,57 @@ describe('ci', () => {
     })
 
     test
-    .stdout()
-    .nock('https://api.heroku.com', api => {
-      api.get(`/pipelines?eq[name]=${pipeline.name}`)
-      .reply(200, [
-        {
-          id: pipeline.id,
-          name: pipeline.name
+      .stdout()
+      .nock('https://api.heroku.com', api => {
+        api.get(`/pipelines?eq[name]=${pipeline.name}`)
+          .reply(200, [
+            {
+              id: pipeline.id,
+              name: pipeline.name
+            }
+          ])
+
+        api.get(`/pipelines/${pipeline.id}/test-runs`)
+          .reply(200, testRuns)
+      })
+      .command(['ci', `--pipeline=${pipeline.name}`])
+      .it('it shows the latest 15 test runs', ({stdout}) => {
+        expect(stdout).to.contain(`=== Showing latest test runs for the ${pipeline.name} pipeline`)
+
+        for (let i = 5; i < 10; i++) {
+          expect(stdout).to.contain(`${statusIcon[i % 4]}  ${testRuns[i].number}   master  ${testRuns[i].commit_sha}  ${testRuns[i].status}\n`)
+        }
+        for (let i = 10; i < 20; i++) {
+          expect(stdout).to.contain(`${statusIcon[i % 4]}  ${testRuns[i].number}  master  ${testRuns[i].commit_sha}  ${testRuns[i].status}\n`)
         }
-      ])
 
-      api.get(`/pipelines/${pipeline.id}/test-runs`)
-      .reply(200, testRuns)
-    })
-    .command(['ci', `--pipeline=${pipeline.name}`])
-    .it('it shows the latest 15 test runs', ({stdout}) => {
-      expect(stdout).to.contain(`=== Showing latest test runs for the ${pipeline.name} pipeline`)
+        expect(stdout).not.to.contain(`${testRuns[4].number} ${testRuns[4].commit_sha}`)
+      })
 
-      for (let i = 5; i < 10; i++) {
-        expect(stdout).to.contain(`${statusIcon[i % 4]}  ${testRuns[i].number}   master  ${testRuns[i].commit_sha}  ${testRuns[i].status}\n`)
-      }
-      for (let i = 10; i < 20; i++) {
-        expect(stdout).to.contain(`${statusIcon[i % 4]}  ${testRuns[i].number}  master  ${testRuns[i].commit_sha}  ${testRuns[i].status}\n`)
-      }
+    test
+      .stdout()
+      .nock('https://api.heroku.com', api => {
+        api.get(`/pipelines?eq[name]=${pipeline.name}`)
+          .reply(200, [
+            {
+              id: pipeline.id,
+              name: pipeline.name
+            }
+          ])
 
-      expect(stdout).not.to.contain(`${testRuns[4].number} ${testRuns[4].commit_sha}`)
-    })
+        api.get(`/pipelines/${pipeline.id}/test-runs`)
+          .reply(200, testRuns)
+      })
+      .command(['ci', '--json', `--pipeline=${pipeline.name}`])
+      .it('it shows the latest 15 test runs in json', ({stdout}) => {
```

Comment:
@talbright no need to say `it` again.

Reply:
This part was actually already there, it's just showing up because vscode corrected the formatting on save. There's a couple of other places where formatting wasn't consistent around `{` and `}`. Let me know how the project prefers to handle whitespace issues.

Reply:
@talbright yeah, I realise the white space changes are revealing this, but while you're there, I figure go ahead and make it consistent.  Here's an example of what I'm referring to:

`.it('shows the latest 15 test runs in json', ({stdout}) => {`